gpu_ocean/SWESimulators/gpu_kernels/CTCS_eta_kernel.cu

```diff
@@ -98,17 +87,17 @@ __global__ void computeEtaKernel(
     }
     
     //Read V into shared memory
-    for (int j=ty; j<block_height+1; j+=blockDim.y) {
+    for (int j=ty; j<block_height+2; j+=blockDim.y) {
         //const int l = clamp(by + j - 1, 0, ny_); // prevent out of bounds
-        const int l = by + j;
+        const int l = by + j - 1;
```

Comment:
I trust that these are correct, but now the kernels are really challenging wrt indices

Reply:
Yes, and indices are updated correspondingly